tests/unit/test_extractor.py

```diff
@@ -0,0 +1,172 @@
+import pytest
+import pandas as pd
+from cognite.client.data_classes import EventWrite
+from cdf_fabric_replicator.extractor import CdfFabricExtractor
+from cdf_fabric_replicator.config import Config
+from cognite.extractorutils.base import CancellationToken
+from cognite.extractorutils.statestore import LocalStateStore
+from cognite.extractorutils.configtools import load_yaml
+
+@pytest.fixture(scope="session")
```

Comment:
Is there a reason you've put the fixtures directly in the test rather than using a `conftest.py` file?  The config could be reused in other unit tests

Reply:
yeah I think we made this initially to mimic the other unit tests but we could move this one specifically to a conftest file

Reply:
I added those to a conftest file to be shared in the future